### PR TITLE
Added Footer

### LINKED ISSUE #1: Add footer to home page
image

### PR DESCRIPTION
**Pull request by rajaFlutter**
Fixed #1

### CODE REVIEW

File: src/app/globals.css [hunk 1]
```diff
@@ -365,6 +365,9 @@ input[type="number"] {
     .gridViewMain-4 {
         @apply grid grid-cols-4;
     }
+    .gridViewMain-4-5 {
+        @apply grid grid-cols-4 grid-rows-5;
+    }
```
Comment (srossross) [inline src/app/globals.css]: https://tailwindcss.com/docs/reusing-styles#avoiding-premature-abstraction

From the tailwinds guide

> Whatever you do, don’t use @apply just to make things look “cleaner”. Yes, HTML templates littered with Tailwind classes are kind of ugly. Making changes in a project that has tons of custom CSS is worse.

> If you start using @apply for everything, you are basically just writing CSS again and throwing away all of the workflow and maintainability advantages Tailwind gives you, for example:

File: src/app/how-it-works/page.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export default function HowItWorksPage() {
+    return (
+        <div className='w-full h-screen flex-row-center-center'>How It works Section</div>
```
Comment (srossross) [inline src/app/how-it-works/page.tsx]: does this conflict with `src/app/how-it-works/page.mxd?`

**Comment by srossross** [comment]
```
colorburst-silence:repos sean$ npm run dev

> repos@0.1.0 dev
> next dev

 ⚠ Port 3000 is in use, trying 3001 instead.
  ▲ Next.js 14.2.5
  - Local:        http://localhost:3001

 ✓ Starting...
   automatically enabled Fast Refresh for 1 custom loader
 ⚠ Duplicate page detected. src/app/how-it-works/page.mdx and src/app/how-it-works/page.tsx resolve to /how-it-works
 ⚠ Duplicate page detected. src/app/how-it-works/page.mdx and src/app/how-it-works/page.tsx resolve to /how-it-works
 ✓ Ready in 651ms
```

**Comment by srossross** [comment]
I'm also getting 

```
Warning: Expected server HTML to contain a matching <div> in <a>.
    at div
    at a
    at LinkComponent (webpack-internal:///(app-pages-browser)/./node_modules/next/dist/client/link.js:121:19)
```

File: components.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (srossross) [inline components.json]: what is this file doing?

File: next.config.js [hunk 4]
```diff
@@ -5,6 +5,10 @@ const nextConfig = {
   // Configure `pageExtensions` to include MDX files
   pageExtensions: ["js", "jsx", "mdx", "ts", "tsx"],
   // Optionally, add any other Next.js config below
+  reactStrictMode: false,
```
Comment (srossross) [inline next.config.js]: turn this to true

File: src/components/ProposalDiv.tsx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+import Link from "next/link";
+import { Progress } from "./shadCDN/progress";
+import { FaImage } from "react-icons/fa6";
+import { HomePageAvatar } from "./HomePageAvater";
+
+
+
+export default function ProposalDiv() {
+    return (
+        <Link href={'/proposals/polars-marco'}
```
Comment (srossross) [inline src/components/ProposalDiv.tsx]: this is hard coded?

File: tailwind.config.ts [hunk 6]
```diff
@@ -1,20 +1,52 @@
...
+const config = {
+  darkMode: ["class"],
   content: [
-    "./src/pages/**/*.{js,ts,jsx,tsx,mdx}",
-    "./src/components/**/*.{js,ts,jsx,tsx,mdx}",
-    "./src/app/**/*.{js,ts,jsx,tsx,mdx}",
+    './pages/**/*.{ts,tsx}',
+    './components/**/*.{ts,tsx}',
+    './app/**/*.{ts,tsx}',
+    './src/**/*.{ts,tsx}',
```
Comment (srossross) [inline tailwind.config.ts]: this changes looks wrong there is a src directory here

File: tailwind.config.ts [hunk 7]
```diff
@@ -1,20 +1,52 @@
...
+          to: { height: "0" },
+        },
+      },
+      animation: {
+        "accordion-down": "accordion-down 0.2s ease-out",
+        "accordion-up": "accordion-up 0.2s ease-out",
       },
     },
   },
-  plugins: [require("@tailwindcss/typography")],
```
Comment (srossross) [inline tailwind.config.ts]: why did you remove this?

**Review by srossross — CHANGES_REQUESTED** [review]
see my other comments. This also breaks the tailwinds "prose"

File: src/app/how-it-works/page.tsx (hunk 2 above)
Comment (rajaFlutter) [reply src/app/how-it-works/page.tsx:5]: No it will not create any conflict.

Comment (rajaFlutter) [reply src/app/how-it-works/page.tsx]: I don't think it would create any conflict. I will check again.

File: src/components/ProposalDiv.tsx (hunk 5 above)
Comment (rajaFlutter) [reply src/components/ProposalDiv.tsx]: yes for now it's hard coded, the fucntion to get the file name from proposal directory is not working properly. I will make this dynamic after fixing that.

File: tailwind.config.ts (hunk 6 above)
Comment (rajaFlutter) [reply tailwind.config.ts]: can you please elaborate like what can be wrong.

File: tailwind.config.ts (hunk 7 above)
Comment (rajaFlutter) [reply tailwind.config.ts]: It was by mistake. I will add it.

File: components.json (hunk 3 above)
Comment (rajaFlutter) [reply components.json]: It's a shadcn configuration file
